aci/resource_aci_fvrsprovcons.go

```diff
@@ -206,6 +218,7 @@ func resourceAciContractProviderCreate(ctx context.Context, d *schema.ResourceDa
 	contractType := d.Get("contract_type").(string)
 
 	ApplicationEPGDn := d.Get("application_epg_dn").(string)
+	MatchT := d.Get("match_t").(string)
```

Comment:
why do this outside the contractType conditional when only used in provider flow